roles/ara_api/tasks/install/podman.yaml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Detecting existing PyPI installation
+  ansible.builtin.stat:
+    path: "{{ ara_api_venv_path }}"
+  register: existing_pypi_install
+
+- name: 'Notify about existing PyPI installation'
+  ansible.builtin.debug:
+    msg: |
+      You seem to have ARA-API installed via PyPI in the past, if you like
+      this role can clean up that installation and migrate your data to the Podman
+      installation, by setting ara_api_migrate_to_podman to true
+  when: existing_pypi_install['stat']['exists']
+
+- name: 'Remove PyPI installation and migrate data'
+  when: ara_api_migrate_to_podman
+  block:
+    - name: Remove PyPI virtualenv
+      ansible.builtin.file:
+        path: "{{ ara_api_venv_path }}"
+        state: absent
+      when: existing_pypi_install['stat']['exists']
+
+    - name: Detecting existing sqlite database
+      ansible.builtin.stat:
+        path: "{{ ara_api_database_name }}"
+      register: existing_sqlite_database
+
+    - name: Move sqlite database to new location
+      ansible.builtin.command: mv {{ ara_api_database_name }} {{ ara_api_root_dir }}/ansible.sqlite
+      when:
+        - existing_sqlite_database['stat']['exists']
+        - ara_api_database_engine == 'django.db.backends.sqlite3'
```

Comment:
I understand the use case for migrating from an installation type to another but I don't know whether we should intend to support it from within the collection, it feels like it will be really hard to support in practice.

In other words, I think this part should live in your (personal) role or playbook, at least for now, anyways.

Otherwise, we'd need to think about how to conceivably provide support for migrating back and forth between the different backends:
- From pypi to podman but also from podman to pypi
- From binary distribution packaging to podman and back
- etc

It's a can of worms :)

Maybe we can leave the warning but supporting the actual migration seems like a headache at this time.

Reply:
Fair enough, sounds sensible, I'll rip it out :)